test/e2e/networking.go

```diff
@@ -241,6 +208,50 @@ var _ = Describe("Networking", func() {
 		Expect(string(body)).To(Equal("pass"))
 	})
 
+	// Marked with [Skipped] to skip the tests by default.  These
```

Comment:
Not much - maybe 15-20s each.  The main cost is downloading the busybox image to the nodes involved.